src/screens/AddNewItemScreen/index.tsx

```diff
@@ -34,14 +38,43 @@ export const AddNewItemScreen: FC = () => {
     navigation.goBack();
   };
 
+  const onConfirmDate = (date: Date) => {
+    onCancelDate();
+    setDate(date);
+  };
+
+  const onCancelDate = () => {
+    setOpen(false);
+  };
+
+  const openDate = () => {
+    setOpen(true);
+  };
+
   return (
     <View style={style.container}>
-      <TextInput
-        style={style.input}
-        placeholder="New task"
-        onChangeText={setText}
-        value={text}
-        selectionColor={COLORS.black}
+      <View style={style.inputContainer}>
+        <CustomInput
+          onChangeText={setText}
+          placeholder={'New task'}
+          value={text}
+          selectionColor={COLORS.black}
```

Comment:
this line is duplicated
maybe it's better to set selectionColor={COLORS.black} in your custom input

Reply:
color may change in the future

Reply:
when it should change then you create this props 
now it is redundant prop

Reply:
ok